masterlist.yaml

```
@@ -21262,13 +21262,30 @@ plugins:
         nav: 1
 
   - name: 'RedBag''s Solitude.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xC025F482
-        util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 95
-        udr: 79
+    url: 
```

Sorry, there are a few more minor corrections I almost missed:

```suggestion
    url:
```